osmaxx/excerptexport/signals.py

```diff
@@ -0,0 +1,21 @@
+from django.core.signals import request_finished
+from django.dispatch import receiver
+
+work_after_request_finished = []
+
+
+def postpone_work_after_request_finished(function, *args, **kwargs):
```

Comment:
This doesn't postpone execution of `function` _after_ the request finishes (i.e. request finishes, _then_ it's postponed), but _until_ the request is finished (i.e., it's postponed _now_, then (sooner or later) request finishes, and then `function` is executed).

So the name should probably be `postpone_work_until_request_finished` or `postpone_work_until_after_request_finished`. (I prefer the former.)